.buildkite/pipeline-scheduler.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+
+'''
+This script is used by schedule-type pipelines
+to automate triggering other pipelines (e.g. Iron Bank validation)
+against release branches
+
+Excepts a (comma separated) env var PIPELINES_TO_TRIGGER.
+An optional EXCLUDE_BRANCHES (comma separated) env var can also be supplied to skip specific branches (e.g. EXCLUDE_BRANCHES="main")
+
+For background info see:
+https://elasticco.atlassian.net/browse/ENGPRD-318 /
+https://github.com/elastic/ingest-dev/issues/2664
+'''
+
+import json
+import os
+import sys
+import time
+import typing
+import urllib.request
+from ruamel.yaml import YAML
+
+
+ACTIVE_BRANCHES_URL = "https://raw.githubusercontent.com/elastic/logstash/main/ci/branches.json"
```

Comment:
Maybe we should change this to the auto-generated one

Reply:
Fixed in https://github.com/elastic/beats/pull/39206/commits/9d0c614413243b121ecf920dff4f62e42fe5e268, great catch